templates/cli/lib/commands/generic.js.twig

```diff
@@ -68,6 +69,13 @@ const client = new Command("client")
             try {
                 let url = new URL(endpoint);
                 if (url.protocol === "http:" || url.protocol === "https:") {
+                    let apClient = new Client().setEndpoint(endpoint);
+                    let response = await apClient.call('GET', '/health/version');
```

Comment:
```suggestion
                    let response = await client.call('GET', '/health/version');
```

Reply:
There is already a `client` defined in this scope.

Reply:
This will override the other `client` object.